compiler/damlc/ide-debug-driver/sample-config.yaml

```diff
@@ -0,0 +1,18 @@
+ide-cmd: daml-head ide --debug
+project-root: /home/moritz/quickstart-java
```

Comment:
Are you sure you want your home directory here?

Reply:
It’s a sample config. Doesn’t really matter what’s in there and my home directory seems as good as `/home/user/` or something like that (and it has the advantage that I don’t need to modify the sample config :grinning:).